content/python/concepts/time-module/terms/time-ctime/time-ctime.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'time.ctime()'
+Description: 'Takes a time, expressed in seconds elapsed since the epoch, and returns a formatted string.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Constants'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `ctime()` function takes a time, expressed in seconds since the epoch, and returns a formatted string.
+
+## Syntax
+
+```pseudo
+ctime(s)
+```
+
+The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00. If no argument is provided the function will use the current local time.
```

Comment:
```suggestion
The argument `s` is the number of seconds since the epoch. The epoch is defined as January 1, 1970, 00:00:00 UTC. If no argument is provided the function will use the current local time.
```